images/oc-build-deploy-dind/openshift-templates/cli/deployment.yml

```diff
@@ -112,6 +112,13 @@ objects:
                   memory: 10Mi
                 limits:
                   memory: 8Gi
+              readinessProbe:
+                initialDelaySeconds: 15
+                exec:
+                  command:
+                  - cat
+                  - /tmp/ready
```

Comment:
I'm a bit worried that this could break existing cli images that don't have that file yet, can we extend this check that it actually checks if the lagoon version in `/lagoon/version` is `1.8.0` and higher and only then we check for the /tmp/ready file?

Reply:
Shouldn't this template only be used on a fresh deploy? Thus the newly running CLI pod would have the file?

Reply:
there is a technical possibility that somebody has v1.7 in their cli images defined and therefore this file never exists which means the test will never pass.
So I think we need to add a check for the lagoon version that this is introduced.